packages/solutions/module-tools/src/builder/esbuild/adapter.ts

```diff
@@ -28,7 +28,7 @@ const globalNamespace = 'globals';
 
 const HTTP_PATTERNS = /^(https?:)?\/\//;
 const DATAURL_PATTERNS = /^data:/;
-const HASH_PATTERNS = /#[^#]+$/;
+const HASH_PATTERNS = /.#[^#]+$/;
```

Comment:
can I delete `HASH_PATTERNS`?

Reply:
Preferably not

Reply:
https://github.com/web-infra-dev/modern.js/blob/04fbf2728cbfa41dccfd92e709f7da88fe1c5d2e/packages/solutions/module-tools/src/builder/feature/style/postcssUrlPlugin.ts#L10  same const here